tests/a_scripts/data_prep/test_split_eval_set.py

```diff
@@ -0,0 +1,217 @@
+import os
+import json
+import pytest
+import hashlib
+from unittest.mock import patch
+
+from llmfoundry.command_utils import split_eval_set_from_args, split_examples
+from llmfoundry.command_utils.data_prep.split_eval_set import DELTA_JSONL_REGEX, get_dataset_format, HF_REGEX, REMOTE_OBJECT_STORE_FILE_REGEX
+
+# Default values
+OUTPUT_DIR = "tmp-split"
+TMPT_DIR = "tmp-t"
+DATA_PATH_SPLIT = "train"
+EVAL_SPLIT_RATIO = 0.1
+DEFAULT_FILE = TMPT_DIR + "/train-00000-of-00001.jsonl"
+
+
+def test_delta_jsonl_regex():
```

Comment:
can you rewrite these tests to use `pytest.mark.parametrize`? e.g. this one would be like
```
@pytest.mark.parametrize("path,expectation", [("tmp-t", True), ("/tmp-t", False))
```